core/mpi/controller/paciente.ts

```diff
@@ -1056,3 +1056,81 @@ EventCore.on('mpi:patient:create', async (patientCreated) => {
     }
 
 });
+export async function buscarRelaciones(dni) {
+    const pac: any = await paciente.findOne(dni);
+    let relaciones = pac.relaciones;
+    let arrayRelaciones: any = [];
+    if (relaciones) {
+        for (let index = 0; index < relaciones.length; index++) {
+            let objetoRelacion: any = new Object();
+            objetoRelacion.nombre = relaciones[index].nombre;
+            objetoRelacion.apellido = relaciones[index].apellido;
+            objetoRelacion.documento = relaciones[index].documento;
+            objetoRelacion.relacion = relaciones[index].relacion;
+            let pacienteRelacion: any = await paciente.findOne({ documento: objetoRelacion.documento });
+            objetoRelacion.id = pacienteRelacion.id;
+            objetoRelacion.edad = calcularEdad(pacienteRelacion.fechaNacimiento);
```

Comment:
se puede usar mongoose directamente para acceder a la edad del paciente, ya que previamente hiciste un findOne.

```suggestion
            objetoRelacion.edad = pacienteRelacion.edad();
```

Reply:
Use el buscarPacByDoc y deje la función que estaba de la edad, lo habíamos visto con Hugo en un commit anterior para unificar la función de la edad y usarla